Units_Engine/Convert/Molality/Molality.cs

```diff
@@ -110,6 +110,8 @@ public static double ToMolality(this double molePerKilograms, object unit)
                     return UNU.MolalityUnit.MolePerGram;
                 case MolalityUnit.MolePerKilogram:
                     return UNU.MolalityUnit.MolePerKilogram;
+                case MolalityUnit.MillimolPerKilogram:
```

Comment:
```suggestion
                case MolalityUnit.MillimolePerKilogram:
```
This didn't build prior to this change.